src/Avatar/Avatar.tsx

```diff
@@ -117,6 +115,9 @@ export const Avatar: RneFunctionComponent<AvatarProps> = ({
 }) => {
   const width =
     typeof size === 'number' ? size : avatarSizes[size] || avatarSizes.small;
+  const ContainerComponent =
```

Comment:
I don't think this is right way of doing it, you just moved default prop within the component which *may* cause performance issues, and still this is unreadable(as per you https://github.com/react-native-elements/react-native-elements/pull/3263#issuecomment-997229483)

It has nothing to do with my Pressable PR, as title of this PR say. You need to take care in autoDoc Parser.

I guess we can use `@defaultValue` property of TSDoc.

@khushal87 @flyingcircle